public/index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
     <title>Home to the creative | Authors Haven</title>
 </head>
+
 <body>
     <div id="root"></div>
 </body>
-</html>
+
+</html>
```

Comment:
Hey @cupidcoder. Great job. But EOL here please.

Reply:
@Chiazokam This has been worked